Talent.Services.Profile/Controllers/ProfileController.cs

```diff
@@ -237,13 +508,64 @@ public ActionResult getProfileImage(string Id)
             //Please do logic for no image available - maybe placeholder would be fine
             return Json(new { profilePath = profileUrl });
         }
+        //public ActionResult getProfileImage(string Id = "")
+        //{
+        //    String talentId = String.IsNullOrWhiteSpace(Id) ? _userAppContext.CurrentUserId : Id;
+        //    var profileUrl = _documentService.GetFileURL(talentId, FileType.ProfilePhoto);
+        //    //Please do logic for no image available - maybe placeholder would be fine
+        //    if (profileUrl == null)
+        //        return Json(new { Success = false, message = "No Image available" });
+        //    return Json(new { profilePath = profileUrl });
+
+        //}
 
         [HttpPost("updateProfilePhoto")]
         [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Roles = "talent")]
+        //public async Task<ActionResult> UpdateProfilePhoto(IFormFile file)
+        //{
+        //    if (file != null)
+        //    {
+        //        try
+        //        {
+        //            await _profileService.UpdateTalentPhoto(_userAppContext.CurrentUserId, file);
+        //            return Json(new { Success = true });
+        //        }
+        //        catch (Exception e)
+        //        {
+        //            return Json(new { Success = false });
+        //        }
+        //    }
+        //    return Json(new { Success = false });
+        //    //Your code here;
+        //    //throw new NotImplementedException();
+        //}
+
         public async Task<ActionResult> UpdateProfilePhoto()
         {
-            //Your code here;
-            throw new NotImplementedException();
+            IFormFile file = Request.Form.Files[0];
```

Comment:
Can you verify if **Request.Form.Files** contains items before you target the index 0